javascript/src/openapi/models/ObjectSerializer.ts

```
@@ -1,251 +1,249 @@
+        return data.toISOString();
+      }
+    } else {
+      if (enumsMap.has(type)) {
+        return data;
+      }
+      if (!typeMap[type]) { // in case we dont know the type
+        return data;
+      }
+
+      // Get the actual type of this object
+      type = this.findCorrectType(data, type);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)